scripts/docker_build.sh

```diff
@@ -20,7 +20,7 @@
 DOCKER_ARGS=$@
 
 GIT_ROOT=$(git rev-parse --show-toplevel)
-GIT_REVISION=$($GIT_ROOT/gradlew -q properties --property version | grep -oP "version: \K.+")
+GIT_REVISION=$($GIT_ROOT/gradlew -q properties --property version |  awk '/version: / {print $2}')
```

Comment:
Please no heavy-weight `awk` either. How about `sed -nr "s/version: (.+)/\1/p"`, does that work on macOS?

Reply:
Works, but i need just disagree about "heavy-weight" awk, when you need entire gradlew and parsing the entire project in the same line before :-)

Reply:
Awk is a new dependency, Gradle is not. If it's in use already anyway, it doesn't matter that much how heavy-weight the tool is. And Gradle we bootstrap, whereas Awk might not be available on all platforms, like Windows (via WSL or Git Bash).